BlueWizard/SpeechSynthesizer.mm

```diff
@@ -4,6 +4,13 @@
 #import "tms5220.h"
 #import "UserSettings.h"
 
+#define SPEAK_EXTERNAL_COMMAND 0x60
+#define RESET_COMMAND 0xFF
+#define SAMPLES_PER_HALF_FRAME (200)
+#define STATUS_TS_MASK 0x80
+#define STATUS_BL_MASK 0x40
+#define STATUS_BE_MASK 0x20
+
```

Comment:
can we please make these use static const instead of #define?

    static int const SPEAK_EXTERNAL_COMMAND = 0x60;
    static int const RESET_COMMAND = 0xFF;
    ... etc